build.gradle

```diff
@@ -44,4 +44,15 @@ patchPluginXml {
     changeNotes """
       Change Notes.<br>
       <em>Maven (single module) and Rust projects supported</em>"""
-}
\ No newline at end of file
+}
+
+/*
+   Requires a clone of https://github.com/JetBrains/intellij-community.
+   
+   Create a gradle.properties file at the root of this project and add the following
+   
+   IDEA_SOURCES_PATH=/path/to/intellij/community/sources
+ */
+test {
+    systemProperty "idea.home.path", "$IDEA_SOURCES_PATH"
```

Comment:
What happens of you dont do this? does it fail in a way that you can detect and fix easily?